src/value.rs

```diff
@@ -1534,10 +1527,10 @@ impl<'a> Set<'a> {
 
 impl<'a> Function<'a> {
     /// Create a function in the current execution context for a given callback.
-    pub fn new(isolate: &'a isolate::Isolate,
+    pub fn new<'b>(isolate: &'a isolate::Isolate,
```

Comment:
I realized that if the method takes a `Box<>`, we are obliged to call the destructor from V8 land, which I'm not sure how to do.